### ISSUE TITLE
NVDA does not announce "Microsoft Dynamics 365 Guides(Preview)" as header.

### ISSUE THREAD

**Issue by Vividha-Khandelwal** [nvaccess/nvda#10194]
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: Microsoft Guides 
Windows Version: Windows10
Desktop application version: 105.0.1909.19
Username/Password: esndev@rubymr.com/Soz****0

Steps to Reproduce:
1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.
2. Sign-In into application using given credentials.​
3. Navigate to left navigation pane and select "About".
4. Start screen reader as Narrator or NVDA.

Actual:
Narrator/NVDA does not announce "Microsoft Dynamics 365 Guides(Preview)" as header.

Expected:
Narrator/NVDA should announce "Microsoft Dynamics 365 Guides(Preview)" as header.

User Impact:
The necessary headings on the screen will not be announced to the screen reader users to understand the proper structure of the content.

### Steps to reproduce:
1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.
2. Sign-In into application using given credentials.​
3. Navigate to left navigation pane and select "About".
4. Start screen reader as NVDA.

### Actual behavior:
NVDA does not announce "Microsoft Dynamics 365 Guides(Preview)" as header.
![23406244_Reactivated_NVDA](https://user-images.githubusercontent.com/48586069/64700030-b8769080-d4c3-11e9-8e1e-c7e40ade1a65.png)

### Expected behavior:
NVDA should announce "Microsoft Dynamics 365 Guides(Preview)" as header.

#### Name and version of other software in use when reproducing the issue:
Microsoft Guides
Desktop application version: 105.0.1909.19

**Comment by Adriani90** [comment]
For this application you need a dedicated account and certain permissions from administrators. Unfortunately it is not possible to reproduce for everyone. Could you please report if this is still occuring with NVDA 2020.1?
When do you exactly expect NVDA to report this as header? When using object navigation? Tabbing around? Using browse mode? Please describe in more details the steps you take to reproduce this issue.

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since more than 3 years, I am closing as abandoned. If you are still having this issue, please comment with an update an requested above and we can reopen.
